functions/bouncer/src/validate/stripe/__tests__/data/customerCreated.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
it looks scrubbed, but just in case it's a, make sure it's scrubbed.

Reply:
Verified scrubbed.